test/testdata/validjson.go

```diff
@@ -0,0 +1,42 @@
+//golangcitest:args -Evalidjson
+package testdata
+
+type Key struct {
+	A, B string
+}
+
+type StrAlias string
+
+type ChanAlias chan<- int
+
+type Textable [2]byte
+
+func (Textable) MarshalText() ([]byte, error) {
+	return nil, nil
+}
+
+func (*Textable) UnmarshalText(_ []byte) error {
+	return nil
+}
+
+type ValidJsonTest struct {
```

Comment:
Maybe we should change the name?
```suggestion
type ValidJSONTest struct {
```